apps/storefront/src/pages/Dashboard/index.tsx

```diff
@@ -104,20 +104,23 @@ function B3Mean({ isMasquerade, handleSelect, startActing, endActing }: B3MeanPr
   );
 }
 
+function useData() {
+  const customerId = useAppSelector(({ company }) => company.customer.id);
+  const b2bId = useAppSelector(({ company }) => company.customer.b2bId);
+  const salesRepCompanyId = +useAppSelector(({ b2bFeatures }) => b2bFeatures.masqueradeCompany.id);
```

Comment:
what do you think to using `Number(whatever)` in new/refactored code?

Reply:
gonna add an eslint rule to change them all 👍